dashboard/app/models/contact_rollups_final.rb

```diff
@@ -15,4 +15,27 @@
 
 class ContactRollupsFinal < ApplicationRecord
   self.table_name = 'contact_rollups_final'
+
+  def self.truncate_and_overwrite
```

Comment:
How about changing `truncate_and_overwrite` to `overwrite_from_processed_table` (truncate and overwrite together seems redundant), and `overwrite_from_processed_table` to `insert_from_processed_table`?